### ISSUE TITLE
Error when installing docker

### ISSUE THREAD

**Issue by mdfh** [hyperledgendary/fabric-devenv#29]
Followed steps as mentioned. Getting the below error after running **vagrant up**, 

![image](https://user-images.githubusercontent.com/22741302/123555558-2bcbbe00-d78f-11eb-87e4-92ad3d97ac1b.png)

1. How to check whether docker is installed or not?
2. How to install the docker in vagrant?

Appreciate your help.

**Comment by jt-nti** [comment]
Hi @mdfh, there seems to be a problem with the bionic64 vagrant box at the moment- I've been getting similar errors...

```
==> default: Updating 'ubuntu/bionic64' with provider 'virtualbox' from version
==> default: '20210609.0.0' to '20210623.0.0'...
==> default: Loading metadata for box 'https://vagrantcloud.com/ubuntu/bionic64'
An error occurred while downloading the remote file. The error
message, if any, is reproduced below. Please fix this error and try
again.

The requested URL returned error: 500
```

I've hit similar problems in the past so I'll investigate switching to the 'hashicorp/bionic64' box instead. In the mean time, you might be able to download the box manually before running `vagrant up` using something like:

```
vagrant box add "https://app.vagrantup.com/ubuntu/boxes/bionic64/versions/20210623.0.0/providers/virtualbox.box" --name "ubuntu/bionic64"
```

**Comment by jt-nti** [comment]
Hopefully this should be fixed in #30 

You shouldn't need to install docker- you can double check using `docker --version` after you SSH into the running Vagrant machine with `vagrant ssh`
